packages/@sanity/cli/src/actions/init-project/initProject.ts

```diff
@@ -963,10 +963,9 @@ export default async function initSanity(
   }
 
   function selectProjectTemplate() {
-    // Make sure the --quickstart and --template are not used together
-    // Force template to clean if --quickstart is used
+    // --quickstart should override template selection
     if (flags.quickstart) {
-      return 'clean'
+      return 'quickstart'
```

Comment:
This is going to break as there is no template called "quickstart". 
The `--quickstart `flag relies on the "clean" template, and modifies the template with its remote schema.